test/distributed/elastic/multiprocessing/api_test.py

```diff
@@ -367,10 +375,11 @@ def test_function_with_tensor(self):
                 pc = start_processes(
                     name="dummy_compute",
                     entrypoint=dummy_compute,
-                    args={},
-                    envs={},
+                    args={0: ()},
+                    envs={0: {}},
```

Comment:
did this behavior change?

Reply:
Its more like a test issue exposed. The PContext class infers nprocs from args size https://fburl.com/630wzww1, so nprocs is 0 before. The non parallel version simply skip starting processes but the parallel version will throw max_workers need to be bigger than 0.